### ISSUE TITLE
could not find sorts

### ISSUE THREAD

**Issue by 20harshita** [kframework/k-legacy#2079]
kjkompile.sh

Preprocessing semantics:
[Error] Compiler: Could not find sorts: [SwitchLabel]
Source(/home/harshita/java-semantics/src/./prep/../common/list-syntax.k)
Location(36,38,36,95)

Execution semantics:
[Error] Critical: Could not find file: builtins/model-checker.k
Lookup directories:[./exec,
/home/harshita/new/k/lib/java/../../include/builtin]
Source(/home/harshita/java-semantics/src/./exec/ltl-support.k)
Location(1,1,1,34)
still producing error

**Comment by daejunpark** [comment]
@20harshita which K version are you using? @laurayuwen could you let him know which version is working for java semantics?

**Comment by laurayuwen** [comment]
The latest K is not working with Java Semantics, sorry. 
Could you try this branch: https://github.com/kframework/k/tree/pldi16 ?

**Comment by laurayuwen** [comment]
Could you try this? https://github.com/kframework/k/tree/javaCompatible

**Comment by 20harshita** [comment]
I M trying..will rply u the result.

**Comment by 20harshita** [comment]
in this screenshot i downloaded TAR.GZ file.

**Comment by 20harshita** [comment]
after runing command kjkompile.sh..will processing semantics take more time
to process??
due to this my system get hanged..shall i have to run it on server??

On Tue, Jan 19, 2016 at 6:49 PM, HARSHITA GOSWAMI <hrshtgoswami339@gmail.com

> wrote:
> 
> I M trying..will rply u the result.

**Comment by laurayuwen** [comment]
Yes, it can take half an hour or more. If your system get hanged, I am
afraid so.

**Comment by 20harshita** [comment]
Can you provide me how to get input from user in K in imp language?
